### DISCUSSION TITLE
Is there a Matrix group or IRC for cockroachDB

### DISCUSSION THREAD

**Discussion by Walexero** [cockroachdb/cockroach#84342] [General]
I would love to contribute to cockroachDB documentation. Is there a group for people focused on the docs and also a group to ask about the technical stuffs when working on the docs for clarity?

**Comment by ajwerner** [comment]
There's a Community Slack. It has channels like #contributors which are a good place to ask technical questions. You can find the sign-up link here: https://www.cockroachlabs.com/join-community/.

**Reply by Walexero** [reply]
Thanks, i appreciate the help
